### PR TITLE
add run type for cron job single/multiple

### PR DESCRIPTION
**Pull request by maheshwari-vaibhav**

### CODE REVIEW

File: src/JobRunner.php [hunk 1]
```diff
@@ -80,6 +80,11 @@ public function run()
                 // How many jobs are runned
                 array_push($this->jobs, $task);
 
+                if (!$task->saveRunningFlag(true) && $task->getRunType() == 'single') {
+                    $this->cliWrite(($task->name ?: 'Task') . ' is single run task and one instance already running.', 'red');
```
Comment (daycry) [inline src/JobRunner.php:84]: $this->cliWrite('Failed: ' . $task->name, 'red');

Comment (maheshwari-vaibhav) [reply src/JobRunner.php]: changes done

Comment (daycry) [reply src/JobRunner.php]: You can't set name of task, the system assign one automatically

Comment (maheshwari-vaibhav) [reply src/JobRunner.php]: https://github.com/daycry/cronjob/pull/20/commits/f9603cdd671709a949c146d1de112e7142dde146

already done in above commit please check

**Comment by daycry** [comment]
Tests failed.

Try to run **composer test** in your local.

**Review by daycry — APPROVED** [review]

**Pull request merged**